BUILDING.md

```diff
@@ -19,7 +19,7 @@ sudo gem update
 
 3) Some dependencies are added via carthage. Run:
 ```
-carthage update
+carthage checkout
```

Comment:
I'm no Carthage expert.  Is that always equivalent, even for people who have just checked out the repo?